Makefile

```
@@ -92,5 +93,4 @@ format-python: .venv/bin/python
 
 .venv/bin/python: Makefile
 	@rm -rf .venv
-	@which virtualenv || sudo pip install virtualenv
-	virtualenv -p $(SYMBOLIC_PYTHON) .venv
+	python3 -m venv .venv
```

Did you intend to use `$SYMBOLIC_PYTHON_VERSION` here?

```suggestion
	$SYMBOLIC_PYTHON_VERSION -m venv .venv
```

---

Yes, I tested it with a hardcoded value before and forgot to change it back >.>